core/server/api_container/server/startosis_engine/interpretation_time_value_store/interpretation_time_value_store.go

```diff
@@ -83,5 +83,6 @@ func (itvs *InterpretationTimeValueStore) GetNewServiceConfig(name service.Servi
 	if !ok {
 		return nil, stacktrace.NewError("Did not find new service config for '%v' in interpretation time value store.", name)
 	}
+	delete(itvs.setServiceConfigValues, name)
```

Comment:
after the new service config values been retrieved, we remove it from the `interpretationTimeValueStore` so that its not used again